packages/fether-electron/src/main/app/constants/index.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// See trustworthiness of 127.0.0.1 vs localhost: https://letsencrypt.org/docs/certificates-for-localhost/
+const TRUSTED_URLS = [
+  'https://127.0.0.1:3000',
```

Comment:
```suggestion
  'http://127.0.0.1:3000',
```

I believe dev mode opens in http

Reply:
I've pushed a commit to fix this

Reply:
when you open dev tools it opens `chrome-devtools://devtools/bundled/toolbox.html?remoteBase=https://chrome-devtools-frontend.appspot.com/serve_file/@b216d5f29ba53b9046287cc7de14f9f0759ad000/&can_dock=true&toolbarColor=rgba(223,223,223,1)&textColor=rgba(0,0,0,1)&experiments=true`.

Do we want to allow this is both dev and prod?